shared/react-native/android/app/src/main/AndroidManifest.xml

```diff
@@ -27,7 +27,7 @@
       <activity
         android:name=".MainActivity"
         android:screenOrientation="portrait"
-        android:launchMode="singleTop"
+        android:launchMode="singleTask"
```

Comment:
i think this is more of what our style of app is. looking at the push issues this seems to make certain flows better. if we see weird android stuff we could revert this